etna/Makefile

```diff
@@ -13,6 +13,7 @@ bash::
 	@ docker-compose run -e SKIP_RUBY_SETUP=1 --rm etna_app bash
 
 release-test::
+	docker network create monoetna_default
```

Comment:
I had to add this step to the `etna` Makefile to create the `monoetna_default` network manually, since it kept failing because the `monoetna_default` network didn't exist. I suspect this is because it's created during some app_fe or edge-apache container, and since etna doesn't have that, the network wasn't being created.

Reply:
:+1: